bookx/production_settings.py

```diff
@@ -0,0 +1,7 @@
+import os
+from .settings import *
+
+DEBUG = False
+SECRET_KEY = os.environ.get("SECRET_KEY", "")
+
+ALLOWED_HOSTS = ["localhost", "138.68.176.181", "bookx.codehub.org.uk"]
```

Comment:
Maybe this can come from the env vars as well?

Reply:
It seems I need to have the list as json in the .env variable then, and in settings I then need to do 
import json
json.loads(os.environ.get("ALLOWED_HOSTS"))
It is probably worth it though. I'll add it to next PR, think that will be easier :)

Reply:
JSON can work. I think comma-separated values, like `localhost,138.68.176.181,bookx.codehub.org.uk` are pretty standard too!
So you could do something like `ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "").split(",")

I've also used [environs](https://pypi.org/project/environs/) before, which does some nice parsing.